packages/eslint-config-eventbrite/rules/best-practices.js

```diff
@@ -1,14 +1,8 @@
 // The rules ultimately override any rules defined in legacy/rules/best-practices.js
 module.exports = {
     rules: {
-        // Require constructor function names to begin with a capital letter
-        // Requires all `new` operators to be called with uppercase-started functions.
-        // Requires all uppercase-started functions to be called with new
-        // http://eslint.org/docs/rules/new-cap
-        'new-cap': 'error',
-
         // disallow modifying properties of parameters
         // http://eslint.org/docs/rules/no-param-reassign
-        'no-param-reassign': ['error', {props: true}]
-    }
+        'no-param-reassign': ['error', {props: true}],
```

Comment:
nit: I would vote that we drop `props`

Reply:
Why? This warns against mutating the properties of parameters as well. We shouldn't be doing that either right?

Reply:
I should have opened that up and read more. I implied it incorrectly. This is perfect!

Reply:
I had to do the same to make sure too!